### PR TITLE
add repository address

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <sunzhu@yunify.com>

### CODE REVIEW

**Comment by soulseen** [comment]
@magicsong @runzexia

File: pkg/docker/docker_test.go [hunk 1]
```diff
@@ -629,3 +629,26 @@ func TestGetImageName(t *testing.T) {
...
 }
+
+func TestGetFullImageName(t *testing.T) {
+	type runtest struct {
+		imageName     string
+		serverAddress string
+		expected string
+	}
+	tests := []runtest{
+		{"test/image","test-harbor.io", "test-harbor.io/test/image:latest"},
```
Comment (runzexia) [inline pkg/docker/docker_test.go]: should add case like `{"test/image","test-harbor.io:3333", "test-harbor.io:3333/test/image:latest"}`

File: pkg/docker/docker.go [hunk 2]
```diff
@@ -739,6 +740,23 @@ func getImageName(name string) string {
 	return name
 }
 
+func GetFullImageName(imageName, serverAddress string) (fullImageName string) {
```
Comment (magicsong) [inline pkg/docker/docker.go]: put this function to package `api` or `run`。Not here

File: pkg/docker/docker_test.go (hunk 1 above)
Comment (soulseen) [reply pkg/docker/docker_test.go]: ok

File: pkg/docker/docker.go (hunk 2 above)
Comment (soulseen) [reply pkg/docker/docker.go]: ok, I put it to `api/types.go`.

**Comment by soulseen** [comment]
@runzexia

**Comment by runzexia** [comment]
need handle merge conficts

**Pull request merged**